packages/components/src/components/Tag/Tag.integ.tsx

```diff
@@ -0,0 +1,19 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { test, expect } from '@playwright/experimental-ct-react';
+
+import { Tag } from './Tag';
+
+test.use({ viewport: { width: 500, height: 500 } });
```

Comment:
Not necessary

Reply:
Since it's a draft PR I am just copy/pasting some example boilerplate while I test the CI

Reply:
I will leave the rest of the boilerplate test code until we do a refactor of the Tag tests to move them to Playwright